static/api/extensions/redhat/rhel-vms/0.10.0/README.md

```
@@ -0,0 +1,170 @@
+### On Windows/WSL/x86_64
+
+The `macadam` binary is embedded in the extension, nothing needs to be installed.
+
+### On Mac/arm64
+
+When initialized, the extension checks if the necessary binaries are present in `/opt/macadam/bin`. Several scenarios can happen:
+- the binary is present and its version is the one expected by the extension (or a later version): the monitoring of VMs starts, and if VMs exist, they are listed in the Settings > RHEL VMs.
+- no binary is present: the monitoring of the VMs is not started, and even if VMs exist, they are not listed in Settings > RHEL VMs. The binary will be installed the first time the user creates a new VM, and the monitoring of VMs will be started at this moment,
+- the binary is present and its version is a prior version than the one expected by the extension: the monitoring is started with the found binary, and if VMs exist, they are listed in Settings > RHEL VMs. The first time the user makes an action on a VM (start, stop, delete, create), the correct binary is installed.
+
+> If this installation fails, you can run the installer manually, using the installer found at https://github.com/crc-org/macadam/releases/tag/v0.3.0. After this, you need to restart the extension which should find and use the binaries.
```

markdownlint-cli2 (0.20.0)

Bare URL used

(MD034, no-bare-urls)